### PR TITLE
[Feature] - ARM support

### PR DESCRIPTION
**Pull request by alefesta**
This PR includes the ARM support leveraging also the `one-click pipeline submit` approach for MLFLOW (Kubeflow is not yet available for ARM architectures).

The PR also includes the usage of MLFLOW image for ARM and k3ai-executor image for ARM.

Code will have to check the architecture to decide what to run and where.
This is supported by #13

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,43 +1,40 @@
...
-	@echo "building ${BIN_NAME} ${VERSION}"
-	@echo "GOPATH=${GOPATH}"
-	GOOS=$(GOOS) GOARCH=$(GOARCH) go build -ldflags "-w -s -X github.com/k3ai/version.GitCommit=${GIT_COMMIT}${GIT_DIRTY} -X github.com/k3ai/version.BuildDate=${BUILD_DATE}" -o bin/${BIN_NAME}
+	go build ${LDFLAGS} -o ${BINARY}
 
-get-deps:
-	dep ensure
+build_all:
+	$(foreach GOOS, $(PLATFORMS),\
+	$(foreach GOARCH, $(ARCHITECTURES), $(shell export GOOS=$(GOOS); export GOARCH=$(GOARCH); go build -v -o bin/$(BINARY)-$(GOOS)-$(GOARCH))))
```
Comment (burntcarrot) [inline Makefile:31]: For releases, we can use [GoReleaser](https://goreleaser.com/).

**Review by burntcarrot — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (alefesta) [reply Makefile:31]: Yeah I would say that for this release I'll still go manual and from next one we will move everything within goreleaser

**Comment by burntcarrot** [comment]
Is this ready to merge?

**Comment by alefesta** [comment]
> Is this ready to merge?

**Almost**, I'll do the last few tests today.. anyway after this release I'll open a new issue related to *non-linux* platforms. Since many tools are not natively supported within ARM we will have to filter the plugins and clusters when a user is not on linux-x86/64 arch.

**Pull request merged**